README.md

```diff
@@ -13,25 +13,31 @@ Supported platforms:
 | x64   | &#x2714; | &#x2714; | &#x2714; |
 | arm64 | &#x2714; |          | &#x2714; |
 
-[1]: https://github.com/apache/parquet-format
-[2]: https://docs.microsoft.com/en-us/cpp/dotnet/how-to-call-native-dlls-from-managed-code-using-pinvoke
-[3]: https://github.com/apache/arrow
-[4]: https://github.com/G-Research/ParquetSharp.DataFrame
-[5]: https://docs.microsoft.com/en-us/dotnet/api/microsoft.data.analysis.dataframe
-
 |                       | Status                                                                                                                                                                                                                         |
 | --------------------: | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | **Release Nuget**     | [![NuGet latest release](https://img.shields.io/nuget/v/ParquetSharp.svg)](https://www.nuget.org/packages/ParquetSharp)                                                                                                        |
 | **Pre-Release Nuget** | [![NuGet latest pre-release](https://img.shields.io/nuget/vpre/ParquetSharp.svg)](https://www.nuget.org/packages/ParquetSharp/absoluteLatest)                                                                                  |
 | **CI Build**          | [![CI Status](https://github.com/G-Research/ParquetSharp/actions/workflows/ci.yml/badge.svg?branch=master&event=push)](https://github.com/G-Research/ParquetSharp/actions/workflows/ci.yml?query=branch%3Amaster+event%3Apush) |
 
+## Why use Parquet?
+
+**Apache Parquet** is an [open source][2], column-oriented data file format designed for efficient data storage and retrieval. It provides efficient data compression and encoding schemes with enhanced performance to handle complex data in bulk. Relative to CSV files, <u>Parquet executes queries **34x faster** while taking up **87% less space**</u>. [Source][7]
+
+[1]: https://parquet.apache.org/
+[2]: https://github.com/apache/parquet-format
+[3]: https://docs.microsoft.com/en-us/cpp/dotnet/how-to-call-native-dlls-from-managed-code-using-pinvoke
+[4]: https://github.com/apache/arrow
+[5]: https://github.com/G-Research/ParquetSharp.DataFrame
+[6]: https://docs.microsoft.com/en-us/dotnet/api/microsoft.data.analysis.dataframe
+[7]: https://towardsdatascience.com/demystifying-the-parquet-file-format-13adb0206705
```

Comment:
These link numbers are now wrong and a lot of the links are pointing to the wrong things. I think the issue is you've changed `[1]` to point to parquet.apache.org instead of the parquet-format repo on GitHub, but rather than delete the original link you've kept it and incremented all the link numbers?

Reply:
Thank you for letting me know! :)